setup_env.py

```diff
@@ -2,10 +2,12 @@
 import logging
 import os
 
-from diem import testnet, chain_ids
+from diem import testnet, LocalAccount, jsonrpc
 
 liquidity_account_name = "liquidity"
 
+PREMAINNET_CHAIN_ID = int(os.getenv("ENV_FILE_NAME", "21"))
```

Comment:
Wait why is this ENV_FILE_NAME?

Reply:
This is what I mean by how default values can be bad -- if there was no default this would fail